test/functional/FunctionalCodeActionsSpec.hs

```diff
@@ -46,7 +48,7 @@ spec = describe "code actions" $ do
       contents <- getDocumentEdit doc
       liftIO $ contents `shouldBe` "main = undefined\nfoo x = x\n"
 
-      noDiagnostics
+      -- noDiagnostics
```

Comment:
Why have these been commented out?

Reply:
No clue. @alanz ? We can try to enable them again.

Reply:
I found the tests were failing on my local machine with them in. I have no strong feeling, so long as CI is happy.

Reply:
lemme try reenabling them